js/index.js

```diff
@@ -1,7 +1,24 @@
 // Iteration 1: Names and Input
+//hacker
+let hacker1 = "Eduardo Brandiao"
+let hacker2 = "Nome nome nome nome"
 
+const hacker1Sentence = "The driver´s name is" + ' ' + hacker1;
```

Comment:
Para os próximos labs, tenta usar o  backtick (`  `) pra ir se acostumando, assim fica mais fácil unir strings com variáveis. Por exemplo: `The driver's name is ${hacker1}`